package.json

```diff
@@ -58,9 +58,9 @@
   },
   "dependencies": {
     "@foxglove/rosmsg-msgs-common": "^1.1.0",
-    "@foxglove/rosmsg-msgs-foxglove": "^2.0.3",
     "@foxglove/rosmsg2-serialization": "^1.0.6",
     "@foxglove/rostime": "^1.1.2",
+    "@foxglove/schemas": "^0.1.2",
```

Comment:
Do we have a timeline for maturing the `@foxglove/schemas` package to 1.0.0?

Reply:
No, but I imagine we will want to wait until Foxglove Studio has full support for the schemas (including forthcoming Marker schemas) so that we know everything is working end-to-end.